.buildkite/dagster-buildkite/dagster_buildkite/utils.py

```diff
@@ -316,6 +316,12 @@ def has_storage_test_fixture_changes():
     )
 
 
+def skip_if_not_dlift_commit():
+    if not message_contains("[dagster-dlift]"):
```

Comment:
could also do a `path.match` condition to check if any of the dbtlift files have been modified